docs/reference/data-streams/data-streams.asciidoc

```diff
@@ -97,12 +97,13 @@ convention:
 
 [source,text]
 ----
-.ds-<data-stream>-<generation>
+.ds-<data-stream>-<yyyy-MM-dd>-<generation>
 ----
 
-Backing indices with a higher generation contain more recent data. For example,
-the `web-server-logs` data stream has a generation of `34`. The stream's most
-recent backing index is named `.ds-web-server-logs-000034`.
+`<yyyy-MM-dd>` is the backing index's creation date. Backing indices with a
```

Comment:
Let me know if you feel `<yyyy-MM-dd>` is clear here. We could also go with `<YYYY-MM-DD>`.

Reply:
Whatever you think is best is fine with me. I used the mixed casing of `<yyyy-MM-dd>` reflexively because it matters in Java, but that probably doesn't apply to user documentation.

Reply:
I'll stick with the current capitalization as it also fits with our standard date formatting: https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-date-format.html